scripts/zones/Bastok_Markets/Zone.lua

```diff
@@ -18,7 +18,8 @@ zoneObject.onZoneIn = function(player, prevZone)
     -- FIRST LOGIN (START CS)
     if player:getPlaytime(false) == 0 then
         if xi.settings.main.NEW_CHARACTER_CUTSCENE == 1 then
-            cs = 0
+            ---@diagnostic disable-next-line: cast-local-type
+            cs = { 0, -1, 0x93 } -- (cs, textTable, Flags)
```

Comment:
Can we make a bitenum for these so they're human readable and can be properly applied to other cutscenes?

Reply:
Where would you like them? I don't know of a lua that makes sense at first glance.

Reply:
One of the interaction Framework luas maybe?

Reply:
Maybe this is a little out of scope for an initial implementation.

0x93 is probably a combination of flags, meaning, we would have to map out which one does what.

Could be done on a separae PR

Reply:
^ I concur on this below. Ill start adding more in following MRs as I go through some other cutscene bugs we have found.